extensions/wikia/AdEngine/js/slot/scrollHandler.js

```diff
@@ -0,0 +1,51 @@
+/*global define*/
+define('ext.wikia.adEngine.slot.scrollHandler', [
+    'ext.wikia.adEngine.adContext',
+    'wikia.log',
+    'wikia.document',
+    'wikia.window'
+], function (adContext, log, doc, win) {
+
+    var logGroup = 'ext.wikia.adEngine.slot.scrollHandler',
+        isRefreshed = {
+            PREFOOTER_LEFT_BOXAD: false,
+            PREFOOTER_RIGHT_BOXAD: false
+        };
+
+    function init() {
+        if (adContext.getContext().opts.enableScrollHandler) {
+            win.addEventListener('scroll', function () {
+                log('Scroll event listener has been added', 'debug', logGroup);
+                for (var slotName in isRefreshed) {
+                    if (isRefreshed.hasOwnProperty(slotName)) {
+                        refreshSlot(slotName);
+                    }
+                }
+            });
+        }
+    }
+
+    function refreshSlot(slotName) {
```

Comment:
We can count here the time a slot was refreshed and pass it to key-value which @nefretete mentioned. What do you think about this idea?